src/Components/Result.js

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable react/prop-types */
+/* eslint-disable react/destructuring-assignment */
+import React from 'react';
+import '../index.css';
+import PropTypes from 'prop-types';
+
+function Result(props) {
+  return (
+    <div>
+      <h2>{props.value}</h2>
+    </div>
+  );
+}
+
+Result.defaultProps = {
+  value: PropTypes.string.isRequired,
+};
```

Comment:
- [ ] the default value of the result prop should be set as 0
Screenshot from pathwright 👇🏾 
<img width="525" alt="Screen Shot 2021-03-23 at 12 45 46 AM" src="https://user-images.githubusercontent.com/48804258/112071945-1c761c80-8b71-11eb-97df-65cd92f83efc.png">